docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -84,13 +84,17 @@ Prebuilt workspaces follow a specific lifecycle from creation through eligibilit
       [`coder_workspace_owner`](https://registry.terraform.io/providers/coder/coder/latest/docs/data-sources/workspace_owner)
       datasources (see [Preventing resource replacement](#preventing-resource-replacement) for further considerations).
 
-   The developer doesn't see the claiming process — the workspace will just be ready faster than usual.
+   The claiming process is transparent to the developer — the workspace will just be ready faster than usual.
 
 You can view available prebuilt workspaces in the **Workspaces** view in the Coder dashboard:
 
 ![A prebuilt workspace in the dashboard](../../../images/admin/templates/extend-templates/prebuilt/prebuilt-workspaces.png)
 _Note the search term `owner:prebuilds`._
 
+Prebuilt workspaces may be operated on like any other workspace. However, if a prebuilt workspace is stopped, the
+reconciliation loop will not destroy it. The intention behind this design is to give operators the ability to park
+problematic prebuilt workspaces in a stopped state for further investigation.
```

Comment:
```suggestion
Prebuilt workspaces can be maintained and updated in the same way as any other workspace.
However, if a Prebuilt workspace is stopped, the reconciliation loop will not destroy it.
This gives template admins the ability to park problematic prebuilt workspaces in a stopped state for further investigation.
```

@dannykopping  by `if a Prebuilt workspace is stopped`, do we mean a not-yet-assigned Prebuilt workspace? (which is my assumption with the first line) Because once it's claimed, it's just a workspace like any other? So what we're talking about here is a (Prebuilt) workspace that is stopped by an admin or by an error?